src/components/contextual/pages/pool/add-liquidity/composables/useMyWallet.ts

```diff
@@ -1,10 +1,10 @@
 import { isSameAddress } from '@/lib/utils';
 import { useTokens } from '@/providers/tokens.provider';
 
-import useInvestPageTabs, {
+import useAddLiquidityTabs, {
```

Comment:
useAddLiquidityPageTabs?

Reply:
I simplified the naming of some of these components, so the component is now `AddLiquidityTabs`